static/src/js/containers/ErrorBannerContainer/ErrorBannerContainer.js

```diff
@@ -21,20 +22,26 @@ export const ErrorBannerContainer = ({ errors, onRemoveError }) => {
   const {
     id,
     message,
-    title
+    title,
+    notificationType
```

Comment:
Rather than preventing the error from being displayed, we should prevent the error from being added to redux within the `addError` action.